desktop/conf/pseudo-distributed.ini.tmpl

```diff
@@ -191,6 +191,9 @@
   # Enable X-Forwarded-Host header if the load balancer requires it.
   ## use_x_forwarded_host=true
 
+  # Enable X-Forwarded-For header if hive/impala requires it.
+  ## use_x_forwarded_for=true
```

Comment:
Same as @ranade1's suggestion to use better config name

Reply:
renamed it to "enable_xff_for_hive_impala"